boa/src/builtins/string/tests.rs

```diff
@@ -36,6 +36,36 @@ fn check_string_constructor_is_function() {
 //     assert_eq!(d, String::from("4"));
 // }
 
+#[test]
+fn new_string_has_length() {
+    let realm = Realm::create();
+    let mut engine = Executor::new(realm);
+    let init = r#"
+        let a = new String("1234");
+        a
+        "#;
+
+    eprintln!("{}", forward(&mut engine, init));
+
+    let empty = String::from("4");
+    assert_eq!(forward(&mut engine, "a.length"), empty);
```

Comment:
We don't need to print everything here, it will slow down testing. If this fails in the future, we can debug then. I also made the final assert easier to read :)
```suggestion
    forward(&mut engine, init);
    assert_eq!(&forward(&mut engine, "a.length"), "4");
```

Reply:
Ahh!  Got it. I was just copying other tests as a template.  I'll put in your suggestions and push again.  Thanks!